client/src/pages/Translations.jsx

```diff
@@ -1,17 +1,112 @@
 // @flow
 
-import React from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import { Tabs, Layout, Row, Table } from 'antd';
 import { Link } from 'react-router-dom';
 import '../css/Translations.css';
 import StatusTag from '../components/StatusTag';
 import PriorityIcon from '../components/PriorityIcon';
+import { getVerifications } from '../utils/api';
 
 const { Header } = Layout;
 const { TabPane } = Tabs;
 
+const languages = ['Spanish', 'French', 'Chinese'];
+const priorities = { Urgent: 4, High: 3, Medium: 2, Low: 1 };
+
 function Translations() {
-  const columns = [
+  const [verifiedList, setVerifiedList] = useState({});
+  const [unverifiedList, setUnverifiedList] = useState({});
+
+  const updateLists = useCallback((verifications) => {
+    const verifiedResources = { Spanish: [], French: [], Chinese: [] };
```

Comment:
might be a future issue but it would be nice if these languages weren't hardcoded if more languages are added in the future